enterprise/frontend/src/metabase-enterprise/whitelabel/components/ChartColorSettings/ChartColorSettings.styled.tsx

```diff
@@ -24,12 +24,12 @@ export const TableTitle = styled.div`
 
 export const TableLink = styled.div`
   display: inline-block;
-  color: ${color("brand")};
+  color: var(--mb-color-brand);
   font-weight: bold;
   cursor: pointer;
 
   &:hover {
-    color: ${darken("brand", 0.12)};
+    color: ${() => darken("brand", 0.12)};
```

Comment:
Note: there is an ongoing discussion about `() => darken("x", y)` versus `({theme}) => darken(theme.fn.themeColor("x"), y)`, see Kelvin's note here https://github.com/metabase/metabase/pull/43512/files#r1627526129

The former uses the `colors` object to get the color, while the latter uses Mantine's theme colors. The former one is terser though. Let's discuss more on that thread.